rllib/policy/tf_policy_template.py

```diff
@@ -62,8 +63,12 @@ def build_tf_policy(
             Policy, ModelV2, TensorType, TensorType, TensorType
         ], Tuple[TensorType, type, List[TensorType]]]] = None,
         mixins: Optional[List[type]] = None,
+        view_requirements_fn: Optional[Callable[[Policy], Dict[
+            str, ViewRequirement]]] = None,
```

Comment:
I thought the only way to specify view reqs would be through custom models. So we should remove this right?

Reply:
Yes, but I still wanted to leave the user some opportunity to add new ones. But it's not needed by any algos right now.